docs/development/workflow/development_workflow.rst

```diff
@@ -251,19 +251,15 @@ Astropy you are working on. Do that with:
 
 .. code-block:: bash
 
-    python setup.py develop  # typically python 2.x, not python 3
-
-or:
-
+    python setup.py install # Where python is python3 
```

Comment:
develop is still preferred here instead of install, and it works well with python3 for a while now.

Reply:
Yup, reverted back to previous state, plus added some comments.

Reply:
well, the rest of the changes were good, my only point was that it should be "python setup.py develop" with python3 and the rest of the notes (and the mention of python setup.py install) can be taken away just as you did previously.

Reply:
But, looking at https://github.com/astropy/astropy/pull/8289/files again, it seems that you're working on an old version and this all has been reworked already.

Reply:
Yeah, I noticed that too not long ago today. But I definitely feel that I should add in a line or two regarding aliasing for `pip=pip3`. Give me some time and see how I could salvage the situation. Am having some mild problems setting up the development version of Astropy on my own machine... I should have it up and running over the next few days.

Reply:
Actually, I think we might need to rework the previously reworked part at PR #8289 . This is because I have just tested out the code... and the following does not work for me:
```
pip install -e .
```
However, the following does, after installing some missing dependency (namely `jinja2`) with pip:
```
python setup.py install # With python 3
```
I think some revising is necessary.

Reply:
BTW, I have used conda/anaconda to create a virtual `env` and have adopted python 3.7 for python (with `python=3.7`).

Reply:
do you have a recent enough version of pip?  
`python setup.py install` is not sufficient as it installs one snapshot that will not track the changes in your local dev version, thus we had `python setup.py develop` there. It didn't use to work with python3, but has been fixed since.

cc @astrofrog for the pip install question.

Reply:
@kakirastern - just to check, what do you mean when you say ``pip install -e .`` doesn't work?

Reply:
@astrofrog Turned out I was missing some important dependency (`cython`) in addition to the `jinja2` I needed to run `pip install -e .[all,test]` successfully. The problem has been fixed just now. I am staring at the relevant Astropy documentation of the installation of source now and realize my error... See [http://docs.astropy.org/en/stable/install.html#building-from-source](http://docs.astropy.org/en/stable/install.html#building-from-source) for my reference. I guess for a rookie like me it takes time to navigate the documentation to get the relevant information.

Reply:
Is it a good idea to include the need to check for dependencies like `cython` and `jinja2` when using the command `pip install -e .[all,test]` in the section "'Activate' the development version of astropy" which is located at [http://docs.astropy.org/en/latest/development/workflow/get_devel_version.html#activate-the-development-version-of-astropy](http://docs.astropy.org/en/latest/development/workflow/get_devel_version.html#activate-the-development-version-of-astropy)?
This is just to serve as a reminder that this way of using `pip` to install the development version of AstroPy is effectively to "build from source", just in case? I think I would have appreciated this if some notes regarding this were in this section as well earlier. (And maybe also that these two dependencies needed to be installed with the proper `pip` version and not with `conda` even though one is working in a virtual python environment?)

Reply:
The way to tell whether it would be good to include is your experience - so, yes, do include it! I see this section very much as aimed at people who have not contributed to another project before.